src/schema/v2/me/delete_account_mutation.ts

```diff
@@ -0,0 +1,82 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLObjectType, GraphQLString, GraphQLUnionType } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { ResolverContext } from "types/graphql"
+
+const AccountMutationDeleteSuccessType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "AccountMutationSuccess",
+  isTypeOf: (data) => {
+    return data._type !== "GravityMutationError"
+  },
+  fields: () => ({
+    success: {
+      type: GraphQLBoolean,
+      resolve: () => true,
+    },
+  }),
+})
+
+const AccountMutationFailureType = new GraphQLObjectType<any, ResolverContext>({
+  name: "AccountMutationFailure",
+  isTypeOf: (data) => {
+    return data._type === "GravityMutationError"
+  },
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => (typeof err.message === "object" ? err.message : err),
+    },
+  }),
+})
+
+export const AccountMutationType = new GraphQLUnionType({
+  name: "AccountMutationType",
+  types: [AccountMutationDeleteSuccessType, AccountMutationFailureType],
+})
+
+export const deleteUserAccountMutation = mutationWithClientMutationId<
+  any,
+  any,
+  ResolverContext
+>({
+  name: "DeleteAccount",
+  description: "Delete User Artsy Account",
+  inputFields: {
+    explanation: {
+      description: "Reason for deleting the account.",
+      type: new GraphQLNonNull(GraphQLString)
```

Comment:
Both these arguments shouldn't be non null as they are optional.

Reply:
@dzucconi What should they be if not non null?